UQ-eLIPSE/crucible-search
# Bugfix/vethub 250

- to test: `yarn test`
- to verify: 
step1: run `yarn dev` at the project root
step2: check any of filter tags, for example:
![image](https://github.com/user-attachments/assets/3f1d58c5-76ca-4986-9e09-a27f1d3d28c1)
step3: unselect a filter tags by click the one in the selected: area, for example:
![image](https://github.com/user-attachments/assets/1b4895ea-8a8f-4e38-ab64-0c7e63ebc601)
